module-order/src/main/java/domain/order_lineitem/OrderLineItem.java

```diff
@@ -1,15 +1,15 @@
-package kitchenpos.domain.order.order_lineitem;
+package domain.order_lineitem;
```

Comment:
자바 패키지는
Package names are written in all lower case to avoid conflict with the names of classes or interfaces.
이런 이유로 모두 소문자로 작성한다고 해요.
관련 [레퍼런스](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html) 남깁니다.2